core/src/test/scala/org/apache/spark/sql/delta/DescribeDeltaDetailSuite.scala

```diff
@@ -58,15 +58,38 @@ trait DescribeDeltaDetailSuiteBase extends QueryTest
       Seq("format", "partitionColumns", "numFiles"))
   }
 
-  test("delta table: path") {
+  test("delta table: Scala details using table path") {
```

Comment:
Can we instead just modify `describeDeltaDetailTest` to check both the SQL command and scala `details()`?